packages/wallet/contract/src/smartWallet.js

```diff
@@ -50,9 +50,13 @@ export const makeSmartWallet = async (
   const notifierParts = {
     contacts: E(admin).getContactsNotifier(),
     dapps: E(admin).getDappsNotifier(),
+    /** all current issuers */
     issuers: E(admin).getIssuersNotifier(),
+    /** recently updated offers */
     offers: E(admin).getOffersNotifier(),
+    /** recently updated payments */
```

Comment:
singular "most recently payment". can still be an array for backwards compatibility and comment might explain that.